sdk/communication/communication-chat/src/chatClient.ts

```diff
@@ -130,14 +113,14 @@ export class ChatClient {
       updatedOptions.idempotencyToken = updatedOptions.idempotencyToken ?? generateUuid();
       const updatedRestModelOptions = mapToCreateChatThreadOptionsRestModel(updatedOptions);
 
-      const { _response, ...result } = await this.client.chat.createChatThread(
+      const result = await this.client.chat.createChatThread(
```

Comment:
I am curious, why was `_response` being removed before?

Reply:
Mentioned in the generated file. It got the http response, but the value was not actually in use.

Reply:
We talked offline, it was not part of the GA version in anticipation of moving to core v2 which is being accomplished by this PR.